### DISCUSSION TITLE
jaywcjlove/reference/46c0eb4/docs/cs

### DISCUSSION THREAD

**Discussion by giscus** [jaywcjlove/reference#502] [Q&A]
# jaywcjlove/reference/46c0eb4/docs/cs

https://raw.githack.com/jaywcjlove/reference/46c0eb4/docs/cs.html

**Comment by fufumo** [comment]


**Reply by Xiao7777777777** [reply]
就你会写？

**Reply by ranqi0927** [reply]
俩rz

**Comment by BobH-Official** [comment]
[判断参数类型并转换类型+校验](https://wangchujiang.com/reference/docs/cs.html#%E5%88%A4%E6%96%AD%E5%8F%82%E6%95%B0%E7%B1%BB%E5%9E%8B%E5%B9%B6%E8%BD%AC%E6%8D%A2%E7%B1%BB%E5%9E%8B%E6%A0%A1%E9%AA%8C) 和 [Switch](https://wangchujiang.com/reference/docs/cs.html#switch) 这两种情况是“模式匹配”，是一个在官方和“[类](https://learn.microsoft.com/zh-cn/dotnet/csharp/fundamentals/types/classes)”、“[继承](https://learn.microsoft.com/zh-cn/dotnet/csharp/fundamentals/object-oriented/inheritance)”在同一级（第三级）的重要内容：[模式匹配概述](https://learn.microsoft.com/zh-cn/dotnet/csharp/fundamentals/functional/pattern-matching)。我建议把“模式匹配”作为一个一级标题。

**Comment by 90su09** [comment]
不是本地吗？
